src/twisted/test/test_threadpool.py

```diff
@@ -227,6 +226,10 @@ class Dumb(object):
         self.assertIsNone(onResultRef())
         self.assertIsNone(resultRef[0]())
 
+    if _PYPY:
+        test_threadCreationArgumentsCallInThreadWithCallback.skip = (
+            "Does not work on PYPY")
```

Comment:
This is because of a gc/weakref difference in PyPy - my thoughts here though are that the bits that are failing here really don't have much to do with threadpool failing on PyPy, which it isn't, so test should probably be reworked to be PyPy compatible.